docs/specs/zk_evm/vm_specification/compiler/instructions/evm/environment.md

```diff
@@ -23,7 +23,7 @@ On how the System Contract is called, see
 is common for Yul and EVMLA representations.
 
 The request to the System Contract is done via the
-[SystemRequest](https://github.com/matter-labs/era-compiler-llvm-context/blob/main/src/eravm/context/function/runtime/system_request.rs)
+[SystemRequest](https://github.com/matter-labs/era-compiler-llvm-context/blob/main/src/eravm/context/function/runtime/runtime_code.rs)
 runtime function.
```

Comment:
Thanks for the contribution, link is broken indeed. But I'm not sure `runtime_code.rs` is the right file. @kelemeno could you please check it?